data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditMessage.java

```diff
@@ -48,14 +50,23 @@
 
     protected static final ObjectMapper MAPPER = new ObjectMapper();
 
+    @JsonProperty("leafResourceId")
+    private final String leafResourceId;  //leafResource Id requested
+
+    @JsonProperty("token")
+    private final String token;  //token for this request
+
     @JsonProperty("userId")
-    protected final String userId; //Unique identifier for the user.
+    protected final @Nullable
+    String userId; //Unique identifier for the user.
```

Comment:
`@Nullable` annotation should be before access modifiers, ditto across the board

```suggestion
    @Nullable
    protected final String userId; // Unique identifier for the user.
```